tests/pr_test.go

```diff
@@ -2,30 +2,50 @@
 package test
 
 import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strings"
 	"testing"
 
+	"github.com/gruntwork-io/terratest/modules/files"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
-const advancedExampleDir = "examples/advanced"
+const basicExampleDir = "examples/basic"
+const existingExampleDir = "examples/existing-instance"
 
-func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
-	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:       t,
-		TerraformDir:  dir,
-		Prefix:        prefix,
-		ResourceGroup: resourceGroup,
+// Current supported regions
+var validRegions = []string{
+	"us-south",
+	"eu-de",
+	"eu-gb",
+	"jp-tok",
+	"au-syd",
+}
+
+func setupOptionsBasicExample(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
```

Comment:
why are you not using `TestOptionsDefaultWithVars` ? If you add resource_tags support you can use this function instead